public/less/style.less

```diff
@@ -7,4 +7,8 @@
 
 .poll-view-result-avatar {
   width: 50px;
+}
+
+.poll-creator {
+  z-index: 11000 !important;
```

Comment:
Like I said above this is to pop the creator over the topic composer instead of behind. I think the poll creator is always opened from the topic composer so it makes sense it should float above. The topic composer has a ```z-index``` of 10000 so had to go a bit higher and set the ```!important``` flag so it overwrites the default bootstrap modal ```z-index``` of 1080.